tools/manifest-templator/manifest-templator.go

```diff
@@ -114,6 +114,10 @@ func main() {
 		GeneratedManifests: make(map[string]string),
 	}
 
+	if featureGates != nil && strings.Contains(*featureGates, "NonRoot") && !strings.Contains(*featureGates, "NonRootExperimental") {
```

Comment:
@davidvossel 
Opened https://github.com/kubevirt/project-infra/pull/1606 and I added this check so we don't break...